src/olympia/bandwagon/forms.py

```diff
@@ -195,43 +196,49 @@ def clean_icon(self):
         icon = self.cleaned_data['icon']
         if not icon:
             return
-        if icon.content_type not in ('image/png', 'image/jpeg'):
+        icon_check = ImageCheck(icon)
+        if (icon.content_type not in ('image/png', 'image/jpeg') or
```

Comment:
Just because I actually googled it… I thought `image/jpg` would be relevant too but indeed only `image/jpeg` is the *only* correct mime-type for JPEG files. So nevermind, all good :laughing: